pkg/pushsync/pushsync.go

```diff
@@ -314,18 +268,22 @@ func (ps *PushSync) PushChunkToClosest(ctx context.Context, ch swarm.Chunk) (r *
 			}
```

Comment:
this tag increment is only relevant for uploader, when this is called via `PushChunkToClosest`.
this contextual info should be inherited in this function or we should get rid of this sent count entirely